adapters/appnexus/params_test.go

```diff
@@ -41,6 +41,7 @@ func TestInvalidParams(t *testing.T) {
 
 var validParams = []string{
 	`{"placement_id":123}`,
+	`{"placement_id":"123"}`,
 	`{"placementId":123,"position":"above"}`,
```

Comment:
Nitpick - Can we also add `{"placementId":"123"}` in validParams

Reply:
thanks @Sonali-More-Xandr, this helped in fixing a small bug as well in bidderparams - https://github.com/prebid/prebid-server/pull/2617/commits/ad93e5a76cfcb96ac14724710eb99c1b36f97230